<?xml version="1.0" encoding="UTF-8"?><testsuite errors="0" failures="25" name="com.gorest.crudtests.UsersDataDrivenTest" skipped="0" tests="25" time="101.51" timestamp="2023-01-10T22:26:18.103Z[Europe/London]"><testcase classname="com.gorest.crudtests.UsersDataDrivenTest" name="Data driven test for adding multiple users to the application"><failure message="1 expectation failed.&#10;Expected status code &lt;201&gt; but was &lt;422&gt;.&#10;" type="java.lang.AssertionError">1 expectation failed.
Expected status code &lt;201&gt; but was &lt;422&gt;.
</failure><system-err>1 expectation failed.
Expected status code &lt;201&gt; but was &lt;422&gt;.
&#13;
io.restassured.internal.ResponseSpecificationImpl.validateResponseIfRequired(ResponseSpecificationImpl.groovy:636)&#13;
io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:122)&#13;
io.restassured.internal.ResponseSpecificationImpl.statusCode(ResponseSpecificationImpl.groovy:130)&#13;
io.restassured.internal.ValidatableResponseOptionsImpl.statusCode(ValidatableResponseOptionsImpl.java:117)&#13;
com.gorest.crudtests.UsersDataDrivenTest.createMultipleUsers(UsersDataDrivenTest.java:31)&#13;
</system-err></testcase></testsuite>